python/plugins/processing/algs/saga/SagaAlgorithmProvider.py

```diff
@@ -91,34 +81,28 @@ def _loadAlgorithms(self):
             ProcessingLog.addToLog(ProcessingLog.LOG_ERROR,
                                    self.tr('Problem with SAGA installation: SAGA was not found or is not correctly installed'))
             return
-        if version not in self.supportedVersions:
-            lastVersion = sorted(self.supportedVersions.keys())[-1]
-            if version > lastVersion:
-                version = lastVersion
-            else:
-                ProcessingLog.addToLog(ProcessingLog.LOG_ERROR,
-                                       self.tr('Problem with SAGA installation: installed SAGA version (%s) is not supported' % version))
-                return
+
+        if version not in ['2.3.0', '2.3.1']:
```

Comment:
@ghtmtt I think there is a bit of a mixed up in the saga release versions. The branches in the saga repo are:
master
release-2-3-0
release-2-3-1
release-2-3-lts (the branch which gives you 2.3.2)
release-3.0.0 (hmmm...not consistent naming...dots and dashes!)

I think this line should change to support 2.3.x and not 2.3.0, 2.3.1 only.